drizzlepac/drizCR.py

```diff
@@ -306,8 +306,8 @@ def _drizCr(sciImage, virtual_outputs, paramDict):
             np.bitwise_and(__dqMask,__crMask,__dqMask)
 
             ####### Create the corr file
-            __corrFile = np.zeros(__inputImage.shape,dtype=__inputImage.dtype)
-            __corrFile = np.where(np.equal(__dqMask,0),__blotData,__inputImage)
+            __corrFile = np.where(__dqMask, __inputImage, __blotData)
+            __corrFile /= scienceChip._conversionFactor
```

Comment:
@stsci-hack Eh, now I see that if I looked a little bit past the `__corrFile`, I could have cleaned up `__corrDQMask` as well (no more `np.equal(__dqMask,0)`)